### ISSUE TITLE
Wordpress tags are not displayed in Mastodon as hashtags when using WordPress Post-Format 

### ISSUE THREAD

**Issue by barning** [Automattic/wordpress-activitypub#776]
### Quick summary

When using in Activity-Object-Type the option WordPress Post-Format, the tags are not displayed as hashtags at least in Mastodon.

### Steps to reproduce

- In Settings switch Activity-Object-Type to WordPress Post-Format
- Write a post with tags
- Check the post with a Mastodon App or Web-App

### What you expected to happen

Tags should be displayed as hashtags

### What actually happened

- Post was made here: https://niklasbarning.de/2024/05/23/gruenes-fenster/
- Follow @perspektive@niklasbarning.de and check the post
- Hashtags are missing

### Impact

All

### Available workarounds?

No but the platform is still usable

**Comment by nickbohle** [comment]
Hi, 

In the settings, WordPress Post-Format is activated. And I've also noticed some inconsistencies in how my posts are displayed on the Mastodon web app. Specifically, when looking at my last three posts, which each have different post formats, I've observed the following:

- The quote post and the standard post both display hashtags correctly.
- However, the image post does not display hashtags at all.

Additionally, it appears that the text associated with the image post is also stripped out. This leads me to wonder if the object type "image" even supports texts and hashtags on Mastodon / the Fediverse.

With the update 2.4.0, I just changed the setting of Activity-Object-Type to:
Note (default) 

Done this, I updated the image post by adding two more hashtags.

As a result, the hashtags are now also displayed on Mastodon. And he toot also includes the title and a link to the post now. The text is still stripped out.
